app/src/components/v-form/form-field-raw-editor.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+	<v-dialog :model-value="showModal" persistent @esc="$emit('cancel')">
+		<v-card>
+			<v-card-title>{{ disabled ? t('view_raw_value') : t('edit_raw_value') }}</v-card-title>
+			<v-card-text>
+				<interface-input-code
+					v-if="type === 'object'"
+					:value="internalValue"
+					:disabled="disabled"
+					:line-number="false"
+					:alt-options="{ gutters: false }"
+					:placeholder="t('enter_raw_value')"
+					language="json"
+					@input="internalValue = $event"
+				/>
+				<interface-system-raw-editor
+					v-else
+					:value="internalValue"
+					:type="type === 'string' ? 'text' : type"
+					:disabled="disabled"
+					:language="type === 'object' ? 'json' : type === 'string' ? 'plaintext' : type"
```

Comment:
This nested ternary is a pretty hard to read. Also I believe the first `type === 'object'` to be unnecessary as the above `v-if="type === 'object'"` already prevents the whole system-raw-editor from rendering if it's an object type

Reply:
also not sure if `type` in this context is going to be compatible with the languages of codemirror here. We might just want to opt for using `plaintext` at all times in the raw editor 🤔 

```suggestion
					language="plaintext"
```